packages/labs/src/components/tag-input/tagInput.tsx

```diff
@@ -23,6 +23,13 @@ import {
 import * as Classes from "../../common/classes";
 
 export interface ITagInputProps extends IProps {
+    /**
+     * If this is set to true, a new tag will be created from a populated input field when
+     * the user either clicks out of the component or presses `enter` on the input.
```

Comment:
"If `true`, `onAdd` will also be invoked when the input loses focus. By default, `onAdd` is only invoked on `enter`."

Reply:
Addressed in https://github.com/palantir/blueprint/pull/1966